ovos_bus_client/util/scheduler.py

```diff
@@ -99,12 +97,12 @@ def __init__(self, bus,
         self.bus.on('mycroft.scheduler.get_event',
                     self.get_event_handler)
 
+        self._running = Event()
         self._stopping = Event()
+        self._stopping.set()
```

Comment:
Is this needed if `autostart`?

Reply:
Probably not since `init` now doesn't return until after the thread has been started, but I don't think it makes much difference to do this here or in an `else` case

Reply:
Depends if anything else could be listening at this point.